lib/cloudtasker/config.rb

```diff
@@ -279,5 +279,15 @@ def server_middleware
       yield @server_middleware if block_given?
       @server_middleware
     end
+
+    #
+    # Return if oidc is enabled. This can be enable to allow cloud tasker to work with google cloud run services that
+    # require authentication. Defaults to false.
+    #
+    # @return [Boolean] Flag to enable oidc.
+    #
+    def oidc_enabled
```

Comment:
`@oidc_enabled || false` doesn't add much value. You can just add `oidc_enabled` as an accessor - similar to `store_payloads_in_redis`.

Reply:
Thanks for the feedback, will make sure to do this change,

Reply:
To answer the first question: I manually tested using one of my rails services ( which can only be called through authenticated requests ) on google cloud run by:
* Adding the library to my rails project fetching it from my forked repository
* Setting the oidc_enable to true in the cloudtasker initializer of my rails project
* Calling an endpoint on the rails service through Postman that triggers a cloudtasker worker

 Although I was able to successfully run the worker from my rails service, I did not create an integration test. Again, thanks for the feedback and I really appreciate the work done on this library.